### DISCUSSION TITLE
Unavailable or inactive

### DISCUSSION THREAD

**Discussion by cbroissa-github** [roleoroleo/yi-hack-Allwinner-v2#425] [Q&A]
Hello,
Often my cameras are unavailable (indisponible in French on the photo) or inactive, sometimes there is no image.
Can I do something to active them ?

Other question after several days sometimes the camera doesn’t work in the integration.
I have to reboot the camera or make again the integration.
It’s a known issue ? I hope for a stable integration. It’s a good solution to often reboot camera (each night perhaps) ?

![60188511-3690-465E-B7B3-B973F54A85D4](https://user-images.githubusercontent.com/83312047/165406805-da7e80fa-e69b-4777-8a19-41c3ffb0099e.jpeg)

Thank you.

**Comment by roleoroleo** [comment]
Are sensors available?
Or only the image is not working.

**Reply by cbroissa-github** [reply]
When I refresh the screen, sometimes the image is ok.
But it’s always displayed ‘Inactive’ on the image.
Is there a test I can do to see if sensors are OK ?
I use MQTT and detection doesn’t work.

**Reply by roleoroleo** [reply]
> When I refresh the screen, sometimes the image is ok.
> But it’s always displayed ‘Inactive’ on the image.

What kind of card is?

> Is there a test I can do to see if sensors are OK ?
> I use MQTT and detection doesn’t work.

You can switch on/off the privacy switch or you can power off/on the cam and check if status is "connected".

**Reply by roleoroleo** [reply]
The label "Inactive" can be fixed adding "state" function to the component.
I will update it.

**Reply by cbroissa-github** [reply]
The card is 'grid'

`type: grid
square: false
columns: 1
cards:
  - type: picture-entity
    entity: camera.camera_portail_cam
  - show_state: true
    show_name: true
    camera_view: auto
    type: picture-entity
    entity: camera.camera_jardin_cam`

With the new integration in HACS, the label is now 'on'.

Other thing, a camera detects a movement, but in historic the entities are always 'non detected'.

**Reply by cbroissa-github** [reply]
Icons  are crossed out, it’s the explanation?

 ![DEB08D8B-0114-47A2-BB89-43F5A1CFC57E](https://user-images.githubusercontent.com/83312047/167622675-bf3f3171-e320-4809-b448-1a382f6486ab.jpeg)

**Reply by roleoroleo** [reply]
Is the binary sensor working?
Mi history is ok:
![immagine](https://user-images.githubusercontent.com/39277388/167640939-391c155a-89f2-43f5-a45f-e0a7d20987ff.png)

**Reply by cbroissa-github** [reply]
No it’s always desactivated status in history, but movement detection is ok, automatisations work.
It’s because I rename the entity ?

**Reply by roleoroleo** [reply]
> It’s because I rename the entity ?

I don't know, I have to check it.

**Reply by roleoroleo** [reply]
It depends on what you change.
If you change the name it's not a problem.
If you change the entity id, maybe...
